include/mbedtls/ecp.h

```diff
@@ -196,6 +196,23 @@ mbedtls_ecp_point;
  * cardinality is denoted by \p N. Our code requires that \p N is an
  * odd prime as mbedtls_ecp_mul() requires an odd number, and
  * mbedtls_ecdsa_sign() requires that it is prime for blinding purposes.
+ * The default implementation only initializes \p A without setting it to the
+ * authentic value for curves with <code>A = -3</code>(SECP256R1, etc), in which
+ * case you need to load and free \p A by yourself when using domain parameters
+ * directly, for example:
+ * \code
+ * mbedtls_ecp_group_init(&grp);
+ * CHECK_RETURN(mbedtls_ecp_group_load(&grp, grp_id));
+ * if (mbedtls_ecp_group_a_is_minus_3(&grp)) {
+ *     CHECK_RETURN(mbedtls_mpi_sub_int(&grp.A, &grp.P, 3);
```

Comment:
No, don't do that, it will make `grp` unusable by `mbedtls_ecp` functions that rely on A. Suggesting instead:
```c
mbedtls_mpi_init(&A);
if (mbedtls_ecp_group_a_is_minus_3(&grp)) {
    CHECK_RETURN(mbedtls_mpi_sub_int(&A, &grp.P, 3);
} else {
    CHECK_RETURN(mbedtls_mpi_copy(&A, &grp.A);
}
```
(with corresponding changes to the cleanup section of course, thanks for including it btw).

Reply:
True. Doing so might introduce problems. At least optimization on A=-3 will no more work.